src/Kafka/KafkaInstanceDrawer/KafkaDetailsTab.tsx

```diff
@@ -42,15 +66,80 @@ export const KafkaDetailsTab: FunctionComponent<KafkaDetailsTabProps> = ({
       <DetailsTabAlert expiryDate={expiryDate} />
       <TextContent>
         <TextList component={TextListVariants.dl}>
+          {isTesting && (
+            <>
+              {renderTextListItem(
+                t("common:type"),
+                instanceType === "eval"
+                  ? t("common:trial")
+                  : t("common:standard")
+              )}
+              {instanceType === "standard" &&
+                renderTextListItem(
+                  t("size"),
```

Comment:
I don't see where this string is defined, but it currently displays as "size". 
Instead it should be "Size" with a capital S.

Reply:
done